src/Service/AppStoreService.php

```diff
@@ -1,7 +1,10 @@
 <?php
 namespace Wheniwork\Feedback\Service;
 
-class AppStoreService
+use Zend\Diactoros\Request;
+use Zend\Diactoros\Uri;
```

Comment:
I wouldn't use Diactoros to do this if you are including Guzzle. Inject `GuzzleHttp\Client` and use that instead.